web/src/routes/poems/__layout.svelte

```diff
@@ -40,62 +42,104 @@
   export let categoriesArr: { title: string; _id: string }[] = []
 
   $: filteredPoems = $session ? filterPoems(poems, $session) : poems
+
+  const NAV_OFFSET = `8rem`
+
+  let scrollY = 0
+  let lastScrollY: number
+
+  let innerWidth = 500
+  let showSideBar = true
+  $: {
+    // if scrollY === lastScrollY, change nothing
+    showSideBar =
+      scrollY === lastScrollY
+        ? showSideBar
+        : scrollY < 500 || scrollY < lastScrollY
```

Comment:
Can we hide the sidebar faster by chaning just this one value. I'm guessing that `< 350` might get us close to the spot where the poem mainImage ends?

Reply:
Yes, that is the number we can change to hide the side bar faster — I'll give 350 a try

Reply:
350 actually works really well. the mainImage is just starting to scroll out of the viewport and then the sidebar goes too.